Tests/test_memory.py

```diff
@@ -18,7 +18,7 @@
 
 from time import clock
 
-from iptest import is_cli, is_cli64, is_mono, run_test, skipUnlessIronPython
+from iptest import is_cli, is_cli64, is_mono, run_test, skipUnlessIronPython, retryOnFailure
```

Comment:
Done. Never noticed they were in alphabetical order. My .NET Core PR probably broke a bunch of them...